npm/design-system/src/components/FileExplorer/FileExplorer.module.scss

```diff
@@ -0,0 +1,73 @@
+@use '../../index.scss' as *;
```

Comment:
```suggestion
@import '../../index.scss';
```

Reply:
No. You want `@use`. It deduplicates imports and has many other benefits. https://css-tricks.com/introducing-sass-modules/#import-files-with-use

Reply:
Oh, how much pain in the ass just to avoid css-in-js :D 
I will never understand this probably

Reply:
Why did we decide not to use css-in-js?